Code/Framework/AzCore/AzCore/StringFunc/StringFunc.h

```diff
@@ -491,7 +489,7 @@ namespace AZ
 
         //////////////////////////////////////////////////////////////////////////
         //! StringFunc::AssetPath Namespace
-        /*! For string functions for support asset path calculations 
+        /*! For string functions for support asset path calculations
```

Comment:
This is actually unrelated to the change I made to the file.
It is showing up because the trailing whitespace was removed.
I'll fix the text though